src/SFA.DAS.RequestApprenticeTraining.Database/Tables/ProviderResponse.sql

```diff
@@ -1,11 +1,15 @@
 ﻿CREATE TABLE [dbo].[ProviderResponse]
 (
 	[Id] UNIQUEIDENTIFIER NOT NULL PRIMARY KEY,
+	[ContactName] NVARCHAR(256) NOT NULL CONSTRAINT [DF_ProviderResponse_ContactName] DEFAULT '',
 	[Email] VARCHAR(256) NOT NULL,
 	[PhoneNumber] VARCHAR(25) NOT NULL,
 	[Website] VARCHAR(256) NOT NULL,
+	[AcknowledgedAt] DATETIME2 NULL,
```

Comment:
This has gotten our of sync with my changes were I have moved these to the link table

Reply:
merged from 2067 latest